### PR TITLE
Remove State from QueueSpec in v1beta1

### PR DESCRIPTION
**Pull request by thandayuthapani**
xref https://github.com/volcano-sh/volcano/pull/693#discussion_r376674111

`Remove State from QueueSpec in v1beta1`

### CODE REVIEW

**Comment by k82cn** [comment]
LGTM, @thandayuthapani , please help to fix CI issue :)

File: pkg/apis/scheduling/v1beta1/types.go [hunk 1]
```diff
@@ -262,9 +262,6 @@ type QueueSpec struct {
 	Weight     int32           `json:"weight,omitempty" protobuf:"bytes,1,opt,name=weight"`
 	Capability v1.ResourceList `json:"capability,omitempty" protobuf:"bytes,2,opt,name=capability"`
 
-	// State controller the status of queue
-	State QueueState `json:"state,omitempty" protobuf:"bytes,3,opt,name=state"`
```
Comment (k82cn) [inline pkg/apis/scheduling/v1beta1/types.go]: Please also update crd yaml to make them align :)

**Comment by thandayuthapani** [comment]
@k82cn Have changed CRD yaml and fixed CI.  Please have a look.

**Comment by k82cn** [comment]
thanks very much for your contribution :)

**Pull request merged**